lib/cmd/fh3/shorturl.js

```diff
@@ -0,0 +1,28 @@
+/* globals i18n */
+var fhreq = require("../../utils/request");
+var common = require("../../common.js");
+
+module.exports = {
+  'desc' : i18n._('Shorten a URL with the henr.ie URL shortener'),
+  'examples' :
+    [{
+      cmd : 'fhc shortenurl --url=<url>',
```

Comment:
@camilamacedo86 may want to rename this file:

Example command here is:
* `fhc shortenurl --url=<url>`
* But to run, because of the filename, you need to run `fhc shorturl --url=<url>`

To fix, I think you just need to `mv shorturl.js shortenurl.js`